### ISSUE TITLE
Loop when inspect 1 item in VStack

### ISSUE THREAD

**Issue by YuantongL** [nalexn/ViewInspector#18]
If I try to makeIterator() on VStack with 1 element
```
VStack {
  Text("Some introduction")
}
```
iterator.next() will always have value, this creates a infinite loop

**Comment by YuantongL** [comment]
Looks like A H/V/ZStack is not considered TupleView in the following function
```
static func viewsInContainer(view: Any) throws -> LazyGroup<Content> {
        let unwrappedContainer = try Inspector.unwrap(content: Content(view))
        guard Inspector.isTupleView(unwrappedContainer.view) else {
            return LazyGroup(count: 1) { _ in unwrappedContainer }
        }
        let tupleViews = try Inspector.attribute(label: "value", value: unwrappedContainer.view)
        let childrenCount = Mirror(reflecting: tupleViews).children.count
        return LazyGroup(count: childrenCount) { index in
            let child = try Inspector.attribute(label: ".\(index)", value: tupleViews)
            return try Inspector.unwrap(content: Content(child))
        }
    }
```
I opened the PR for this fix, but idk if this is the correct fix

**Comment by nrivard** [comment]
I have been using `underestimatedCount`:

For example:
```swift
        for index in 0..<forEachInspection.underestimatedCount {
            XCTAssertNoThrow(try forEachInspection.view(BarcodeDigitView.self, index))
        }
```

**Comment by nalexn** [comment]
@YuantongL I've added a test and merged your PR. Nice find!

**Comment by nalexn** [comment]
Fixed in `v0.3.8`
